docs/source/en/using-diffusers/controlling_generation.mdx

```diff
@@ -27,14 +27,15 @@ Depending on the use case, one should choose a technique accordingly. In many ca
 Unless otherwise mentioned, these are techniques that work with existing models and don't require their own weights.
 
 1. [Instruct Pix2Pix](#instruct-pix2pix)
-2. [Pix2Pix Zero](#pix2pixzero)
-3. [Attend and Excite](#attend-and-excite)
-4. [Semantic Guidance](#semantic-guidance)
-5. [Self-attention Guidance](#self-attention-guidance)
-6. [Depth2Image](#depth2image)
-7. [MultiDiffusion Panorama](#multidiffusion-panorama)
-8. [DreamBooth](#dreambooth)
-9. [Textual Inversion](#textual-inversion)
+2. [Pix2Pix 0](#pix2pixzero)
+3. [Attend and excite](#attend-and-excite)
+4. [Semantic guidance](#semantic-guidance)
+5. [Self attention guidance](#self-attention-guidance)
+6. [Depth2image](#depth2image)
+7. [DreamBooth](#dreambooth)
+8. [Textual Inversion](#textual-inversion)
+10. [MultiDiffusion Panorama](#panorama)
```

Comment:
Why were these changes?

Reply:
I think so. Shall I set all of them to "1."?